Meadowlark-js/package.json

```diff
@@ -23,36 +23,38 @@
     "@elastic/elasticsearch": "7.13.0",
     "@hapi/joi": "^17.1.1",
     "aws4": "^1.11.0",
-    "axios": "^0.24.0",
+    "axios": "^0.25.0",
     "deep-freeze": "^0.0.1",
     "didyoumean2": "^5.0.0",
     "ed-fi-model-3.3b": "^2.0.2",
     "fast-memoize": "^2.5.2",
+    "follow-redirects": "^1.14.7",
     "fs-extra": "^10.0.0",
     "inflection": "^1.13.1",
+    "inquirer": "^8.2.0",
     "invariant": "^2.2.4",
     "jssha": "^3.2.0",
-    "metaed-core": "2.6.0",
-    "metaed-plugin-edfi-unified": "2.6.0",
+    "metaed-core": "^2.6.0",
```

Comment:
I'm 50-50 on not locking these to a specific version, but ok for now given the next bump is to 3.0.0